### ISSUE TITLE
[Feature request] Wallpaper(ish) mode

### ISSUE THREAD

**Issue by gaussandhisgun** [NickvisionApps/Cavalier#160]
It would be cool if we were able to use Cavalier as some sort of a live wallpaper, so it renders as a cool flashy animation behind everything else. Here's an example application that already does this in a similar environment: [Hidamari](https://github.com/jeffshee/hidamari) lets you use video wallpapers on Gnome. Maybe you could be inspired by some of their techniques or whatever.

**Comment by I21b** [comment]
This will be very cool, if we can set the Cavalier window to the bottom

**Comment by Anomalocaridid** [comment]
If you use a wayland compositor, https://gitlab.freedesktop.org/mstoeckl/windowtolayer seems to work pretty well.

If you use an x11 window manager, https://github.com/mmhobi7/xwinwrap might work, but I haven't tried it.

**Comment by gaussandhisgun** [comment]
> If you use a wayland compositor, https://gitlab.freedesktop.org/mstoeckl/windowtolayer seems to work pretty well.

Yeah, except i'm on Gnome and Gnome does not support... any shenanigans, really, not even grim | slurp. Its Gnome.
